stage1/stop_kvm/stop.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//+build linux
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"syscall"
+)
+
+const lkvmBinPath string = "stage1/rootfs/lkvm"
+
+func stop() int {
+	pwd, err := os.Getwd()
```

Comment:
Drive by comment: this seems a bit fragile. Maybe we should add a uuid arg to the entrypoint..?